Configurations/ConfigFramework.xcconfig

```diff
@@ -1,8 +1,8 @@
 // Framework only
 
 DYLIB_INSTALL_NAME_BASE = @rpath
-DYLIB_COMPATIBILITY_VERSION = 1.6
-DYLIB_CURRENT_VERSION = $(SPARKLE_VERSION_MAJOR).$(SPARKLE_VERSION_MINOR).$(SPARKLE_VERSION_PATCH)
+DYLIB_COMPATIBILITY_VERSION = $(SPARKLE_VERSION_MAJOR).0.0
```

Comment:
We should not change the compatibility version. On several occasions I debug apps built using Sparkle 1 with  using Sparkle 2 (dyld inserting the framework or otherwise). This will likely break that. Also Sparkle 2 maintains ABI compatibility with Sparkle 1 versions.

Reply:
Right, that's very interesting. And brave 😂 

I've reverted this change in 26b8546d8be0df7e4dd1d1fa1a650215812f653d.